### PR TITLE
[FIX #1721] Disable sign-in button when no password has been entered

### LINKED ISSUE #1721: Login button is enabled for empty password field
### Description

*Type*: Bug

*Summary*: On login screen the Login button is enabled even nothing is entered in password field

#### Expected behavior
Login button is disabled until at least 1 symbol is entered in password field

### Reproduction

- Open Status
- Create account
- Click switch users
- Select account
- Pay attention to Login button

screenshot_2017-08-31-13-30-45-926_im status ethereum 1

### Additional Information

* Operating System: Android, iOS

### PR DESCRIPTION
**Pull request by jeluard**
### Steps to test:
- Open Status
- Login button is disabled when no password has been entered
- Login button is enabled when a password of at least 1 character has been entered

status: ready

### CODE REVIEW

File: src/status_im/ui/screens/accounts/login/styles.cljs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 (ns status-im.ui.screens.accounts.login.styles
   (:require-macros [status-im.utils.styles :refer [defnstyle defstyle]])
-  (:require [status-im.components.styles :as common]))
+  (:require [status-im.components.styles :as st]))
```
Comment (flexsurfer) [inline src/status_im/ui/screens/accounts/login/styles.cljs:3]: ?

File: src/status_im/ui/screens/accounts/login/views.cljs [hunk 2]
```diff
@@ -61,10 +59,11 @@
                                :auto-focus        true
                                :secure-text-entry true
                                :error             (when (pos? (count error)) (i18n/label (error-key error)))}]]
-      [view {:margin-top 16}
-       [touchable-highlight {:on-press #(login-account password-text-input address password)}
-        [view st/sign-in-button
-         [text {:style st/sign-it-text} (i18n/label :t/sign-in)]]]]]
+      (let [enabled? (not (zero? (count password)))]
```
Comment (rasom) [inline src/status_im/ui/screens/accounts/login/views.cljs]: `(pos? (count password))`

**Review by rasom — APPROVED** [review]

File: src/status_im/ui/screens/accounts/login/views.cljs (hunk 2 above)
Comment (jeluard) [reply src/status_im/ui/screens/accounts/login/views.cljs]: Done

**Comment by asemiankevich** [comment]
Verified as fixed, login button is disabled when no password entered in password field.

File: src/status_im/ui/screens/accounts/login/styles.cljs (hunk 1 above)
Comment (flexsurfer) [reply src/status_im/ui/screens/accounts/login/styles.cljs:3]: as i know we're using common for common styles

Comment (flexsurfer) [reply src/status_im/ui/screens/accounts/login/styles.cljs:3]: at least it should be styles

Comment (jeluard) [reply src/status_im/ui/screens/accounts/login/styles.cljs:3]: `common` is not part of the ns so it is a little weird.

For ns we use frequently I think it makes sense to have a shorter alias but we should agree on them.

Comment (flexsurfer) [reply src/status_im/ui/screens/accounts/login/styles.cljs:3]: agree `common` is weird. as i know we decided to use full names for example `styles`

Comment (jeluard) [reply src/status_im/ui/screens/accounts/login/styles.cljs:3]: ok

**Pull request merged**